app/src/organisms/QuickTransferFlow/utils/getFlowRateRange.ts

```diff
@@ -0,0 +1,28 @@
+import { getPipetteNameSpecs } from '@opentrons/shared-data'
+import type { PipetteName } from '@opentrons/shared-data'
+import type { FlowRateKind, QuickTransferSummaryState } from '../types'
+
+export function getFlowRateRange(
+  state: QuickTransferSummaryState,
+  kind: FlowRateKind
+): { min: number; max: number } {
+  let pipetteName = state.pipette.model
+  if (state.pipette.channels === 1) {
+    pipetteName = pipetteName + `_single_flex`
+  } else if (state.pipette.channels === 8) {
+    pipetteName = pipetteName + `_multi_flex`
+  } else {
+    pipetteName = pipetteName + `_96`
+  }
+
+  const spec = getPipetteNameSpecs(pipetteName as PipetteName)
+  let defaultFlowRate
```

Comment:
nit: can you initialize this to 0 and type it as number maybe?

```suggestion
  let defaultFlowRate: number = 0
```

Reply:
This is not a number, it's an object like so `spec?.defaultAspirateFlowRate`
what is the best way to type this interface?

Reply:
oh i see, i recommend making a new type interface at the top of the file and using that. For example:

```
interface DefaultFlowRate {
    default: number
    valuesByApiLevel: Record<string, number>
}
```